src/assignment/graphql.ts

```diff
@@ -0,0 +1,93 @@
+import { gql } from 'apollo-boost';
+import { ITEMS_PER_PAGE } from '../my-workspace/constants';
+
+export const GET_ASSIGNMENT_BY_ID = gql`
+	query getAssignmentsById($id: Int!) {
```

Comment:
Zou er geen manier zijn om deze returning fields in een variable op te slaan en mee te geven want is eigenlijk voor deze 2 hetzelfde. Lijkt me wel dat graphql hier iets voor heeft.

Reply:
Voor het overzicht moeten ik de fields nog limiteren tot enkel die die ik nodig heb. Zodat we minder data ophalen.